src/main/java/com/kodilla/ecommercee/domain/Product.java

```diff
@@ -0,0 +1,53 @@
+package com.kodilla.ecommercee.domain;
+
+import lombok.*;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.math.BigDecimal;
+
+@Getter
+@Setter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table(name = "products")
+public class Product {
+
+    @Column(name = "id")
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+
+    @Column(name = "name")
+    @NotNull
+    private String name;
+
+    @Column(name = "price")
+    @NotNull
+    private BigDecimal price;
+
+    @Column(name = "description")
+    private String description;
+
+    @ManyToOne
+    @JoinColumn(name = "group_id")
+    private Group groupId;
+
+    @ManyToOne
+    @JoinColumn(name = "cart_id")
+    private Cart cartId;
```

Comment:
specyfikacja nie przewiduje pola cartId w klasie Product:
{
    "id": 1,
    "name": "kurtka zimowa",
    "description": "Pellentesque tempus interdum quam ut rhoncus. Donec ullamcorper turpis dolor. Donec euismod pretium eros et eleifend. Aliquam vulputate faucibus lorem non auctor. Vivamus erat turpis, molestie a nisl non, scelerisque luctus enim. Nunc mi mi, laoreet ac mollis nec, pharetra sit amet tortor. Vivamus a bibendum purus.",
    "price": 100,
    "groupId": "1"
}

Reply:
To nawet nie ma żadnego logicznego sensu żeby product posiadał takie pole.